app/editor/src/features/content/tool-bar/ContentToolBar.tsx

```diff
@@ -20,23 +21,46 @@ export interface IContentToolBarProps {
  * @returns A content filter toolbar
  */
 export const ContentToolBar: React.FC<IContentToolBarProps> = ({ onSearch }) => {
-  const [, { storeFilter, storeFilterAdvanced }] = useContent();
+  const [{ filter, filterAdvanced }, { storeFilter, storeFilterAdvanced }] = useContent();
+  const search = fromQueryString(window.location.search);
+  // extract every value from window.location.search that is productIds
+  // and put into an array
+  const extractProductIds = (search: string) => {
```

Comment:
You now have two places doing the exact same complex code that will throw an error if anything is out of place.  Use a utility function that uses existing features in javascript `URLSearchParams.getAll()`.